airbyte-integrations/connectors/source-slack-singer/source_slack_singer/spec.json

```diff
@@ -4,13 +4,7 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Source Slack Api Singer Spec",
     "type": "object",
-    "required": [
-      "start_date",
-      "token",
-      "private_channels",
-      "join_public_channels",
-      "exclude_archived"
```

Comment:
it seems like these toggles don't actually do anything. the slack source can only replicate messages from non-archived, public channels that is has previously been added to. remove these for now until they can be fixed.

talked with @sherifnada a bit. this was a very early source, before we set up our heuristic of only using singer taps that were well maintained. this singer tap gets very little love. likely instead of trying to apply best practices to it, we will want to rewrite our own.

Reply:
Should this PR bump the version as well?

Reply:
yup. added it to the checklist.